server.js

```diff
@@ -58,19 +62,60 @@ MongoClient.connect('mongodb://localhost:4201/tutorial-builder', (err, database)
 SERVER Routes
 
 */
+app.put('/changepass', (req, res) => {
+	db.collection('users').findOne({username: req.body.username}).then(user => {
```

Comment:
`app.put('/changepass', jwtMiddleware({secret: passphrase}), (req, res) => {...}`
this will give you access to the jwt, which has a username field. Access it like `req.user //{username: ..., id: ...}`